tutorial/gui/pcl-visualizer/ClassUsingPclVisualizer.cpp

```diff
@@ -0,0 +1,204 @@
+//! \example ClassUsingPclVisualizer.cpp
+#include "ClassUsingPclVisualizer.h"
+
+#if defined(VISP_HAVE_PCL)
+// PCL
+#include <pcl/io/pcd_io.h>
+
+// Visp
+#include <visp3/core/vpTime.h>
+#include <visp3/core/vpGaussRand.h>
+#include <visp3/core/vpRobust.h>
+#include <visp3/gui/vpColorBlindFriendlyPalette.h>
+#include <visp3/io/vpKeyboard.h>
+
+//! [Z coordinates computation]
+double zFunction(const double &x, const double &y, const unsigned int order)
+{
+  const double offset(0.5);
+  double z(0.);
+
+  for(unsigned int n = 0; n <= order; n++)
+  {
+    for(unsigned int k = 0; k <= order - n; k++)
+    {
+      if(k + n > 0)
+      {
+        z += std::pow(x, n) * std::pow(y, k);
+      }
+      else
+      {
+        z += offset;
+      }
+
```

Comment:
Remove empty line

Reply:
Done in [91391b3](https://github.com/lagadic/visp/pull/1163/commits/91391b324ed48da41cfb5034f2807dc2e77fa549)